server/src/middlewares/index.js

```diff
@@ -0,0 +1,61 @@
+const { channel } = require('./channels');
+const { agentsChannels } = require('./agents-channels');
+const { measurable } = require('./measurables');
+const { measurement } = require('./measurements');
+const { serie } = require('./series');
+
+const middlewares = {
+  Query: {
+
+    // @tested
+    measurements: async (resolve, root, args, context, info) => {
+      await measurable(root, args, context, info);
```

Comment:
This seems a bit awkward to me (though kudos for getting it to this stage). Do we really need to pass all 4 parameters into each function?

Reply:
Yes, we do.

Reply:
If one function will take "root, info", another "args, context" it would be harder to remember signature.

Reply:
Ok, no need to take further action now.